directory-listing/common.css

```diff
@@ -1,60 +1,105 @@
 *, *::before, *::after {
-  box-sizing: border-box;
+    box-sizing: border-box;
 }
 
 html,
 body {
-  margin: 0;
-  color: #263238;
-  font-size: 16px;
-  font-family: system-ui, sans-serif;
-  line-height: 1.5;
+    margin: 0;
+    color: #212121;
+    font-size: 16px;
+    font-family: system-ui, sans-serif;
+    line-height: 1.5;
+    text-align: center;
 }
 
 body {
-  min-height: 100vh;
-  border-top: 0.5rem solid #607d8b;
-  border-bottom: 0.5rem solid #607d8b;
+    min-height: 100vh;
+    border-top: 0.5rem solid #90a4ae;
+    border-bottom: 0.5rem solid #90a4ae;
 }
 
-main {
+h1{
+  width: 95%;
   margin: 0 auto;
-  padding: 2rem 1rem;
-  max-width: 56rem;
+  margin-top: 1rem;
+  font-family: monospace;
+}
+
+table {
+    text-align: left;
+    white-space: nowrap;
+    border-collapse: collapse;
+    width: 95%;
+    margin: 0 auto;
+}
+
+table tr:hover {
+    background-color: #eee;
+}
+
+table tr .indexcolicon a {
+    border: none;
+}
+
+main {
+    margin: 0 auto;
+    padding: 2rem 1rem;
+    max-width: 56rem;
 }
 
 a {
-  color: #0d47a1;
-  display: inline-block;
-  text-decoration: none;
-  border-bottom: 1px solid transparent;
+    color: #303f9f;
+    display: inline-block;
+    text-decoration: none;
+    border-bottom: 1px solid transparent;
 }
 
 a:hover {
-  border-bottom: 1px solid currentColor;
+    border-bottom: 1px solid currentColor;
 }
 
 .title {
-  font-size: 1.25rem;
-  font-weight: 500;
-  margin: 2rem 0;
+    font-size: 1.25rem;
+    font-weight: 500;
+    margin: 2rem 0;
 }
 
 .title .js-path a {
-  padding: 0 0.25rem 0;
-  text-decoration: none;
+    padding: 0 0.25rem 0;
+    text-decoration: none;
 }
 
 .title .js-path a:hover {
-  opacity: 1;
+    opacity: 1;
 }
 
 footer {
-  padding: 2rem 1rem;
-  text-align: center;
-  font-size: 0.875rem;
+    padding: 2rem 1rem;
+    text-align: center;
+    font-size: 0.875rem;
 }
 
 footer a {
-  color: #546e7a;
+    color: #607d8b;
+}
+
+.even-parentdir .indexcolname a {
+    position: relative;
+    color: transparent;
+}
+
+.even-parentdir .indexcolname a::after {
+    content: '..';
```

Comment:
I don't like this, changing "Parent Directory" to "..". Some people may understand what it is but its particularly vague.